What does the following error message tell you?

ArgumentError: wrong number of arguments (1 for 2)
  from (irb):1:in `calculate_product'
  from (irb):4
  from /Users/username/.rvm/rubies/ruby-2.0.0-p353/bin/irb:12:in `<main>'

ANSWER:

This says that on line 4 of the program the 'calculate_product' method was invoked
and only 1 argument was passed into the method call when 2 arguments are expected.